library/src/commonMain/kotlin/ovh/plrapps/mapcompose/api/LayerApi.kt

```diff
@@ -114,6 +114,31 @@ fun MapState.moveLayerDown(layerId: String) {
     }
 }
 
+/**
+ * Pop the top layer from the stack.
+ */
+fun MapState.popLayer() {
```

Comment:
I suggest renaming the function to `removeLastLayer` (to keep consistency with existing apis)